ui/plugin-system/src/components/OptionsEditorTabs/OptionsEditorTabs.test.tsx

```diff
@@ -103,59 +91,14 @@ describe('OptionsEditorTabs', () => {
     expect(activeTabPanel).toHaveTextContent('settings configuration');
   });
 
-  it('renders custom tabs between visual tabs and json editor', () => {
-    renderCustomTabs();
+  it('renders tabs in correct order', () => {
+    renderTabs();
 
     const tabList = screen.getByRole('tablist');
     const tabs = getAllByRole(tabList, 'tab');
-    expect(tabs).toHaveLength(5);
+    expect(tabs).toHaveLength(3);
     expect(tabs[0]).toHaveTextContent('Query');
     expect(tabs[1]).toHaveTextContent('Settings');
-    expect(tabs[2]).toHaveTextContent('Table column');
-    expect(tabs[3]).toHaveTextContent('Table options');
-    expect(tabs[4]).toHaveTextContent('JSON');
-  });
-
-  it('shows the correct content when selecting a custom tab', () => {
```

Comment:
these unit tests are not needed anymore now that the component doesn't have different types of tabs